base/build.gradle

```diff
@@ -40,6 +40,8 @@ android {
         abortOnError false
         lintConfig file('lint.xml')
     }
+
+    //preBuild.dependsOn 'checkstyle'
```

Comment:
Altho checkstyle plugin should already add dependency to `check` task. Please verify that.